static/docs/user-guide/dvc-file-format.md

```diff
@@ -45,30 +45,30 @@ locked: True
 
 On the top level, `.dvc` file consists of such fields:
 
-- `cmd`: a command that is being run in this stage;
-- `deps`: a list of dependencies for this stage;
-- `outs`: a list of outputs for this stage;
-- `md5`: md5 checksum for this DVC-file;
-- `locked`: whether or not this stage is locked from reproduction;
-- `wdir`: a directory to run command in (default `.`);
+- `cmd`: a command that is being run in this stage.
+- `deps`: a list of dependencies for this stage.
+- `outs`: a list of outputs for this stage.
+- `md5`: md5 checksum for this DVC-file.
+- `locked`: whether or not this stage is locked from reproduction.
+- `wdir`: a directory to run command in (default `.`).
```

Comment:
I think that none of the bullets in this doc should have no punctuation at the end (if we agree on the rules I suggested in https://github.com/iterative/dvc.org/issues/491#issuecomment-514118621), as I changed in [34df9cd @ dvc-file-format.md](https://github.com/iterative/dvc.org/pull/490/commits/34df9cd08bab0d34213470b398a2d931e4069f67#diff-a833a0a32123749893e6d04ce58f3834) (pending merge which may take a while so go feel free to apply here again anyway).

Reply:
I have already removed them.